internal/api/grpc/server/middleware/translator.go

```diff
@@ -40,13 +40,13 @@ func translateError(ctx context.Context, err error, translator *i18n.Translator)
 	return err
 }
 
-func newZitadelTranslator(defaultLanguage language.Tag) (*i18n.Translator, error) {
-	return translatorFromNamespace("zitadel", defaultLanguage)
+func newZitadelTranslator(defaultLanguage language.Tag, allowedLanguages []language.Tag) (*i18n.Translator, error) {
```

Comment:
do we have multiple definitions of "newZitadelTranslator" throughout the code?

Reply:
What do you mean with multiple definitions?

Reply:
I centralized access to language filesystems and translator creation :+1: